deltacat/compute/compactor_v2/private/compaction_utils.py

```diff
@@ -585,7 +585,9 @@ def _process_merge_results(
             f"as or greater than params.num_rounds, which is {params.num_rounds}"
         )
         hb_id_to_entry_indices_range[str(mat_result.task_index)] = (
-            file_index,
+            hb_id_to_entry_indices_range.get(str(mat_result.task_index), [file_index])[
```

Comment:
Could you add a comment explaining what this line is doing?

Reply:
done